.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Bug report
-about: Report a Gson bug.
+about: Report a Gson bug. Please have a look at the troubleshooting guide (Troubleshooting.md) first.
```

Comment:
Have used plain text here instead of a proper link to the guide because apparently this 'about' text does not support Markdown.